### PR TITLE
fix(theme): fixed bugs related to buttons flair and text

### PR DESCRIPTION
**Pull request by ThomBos**
**What this PR does** 📖
Fixes theme issues with the buttons and reorders the code, every color-related thing is now in a unique style file

**Which issue(s) this PR fixes** 🔨
AP-414 and AP-428 

**Special notes for reviewers** 🗒️
Created a new "satellite" flair and set it as default, so is no longer necessary locking the flair select

**Additional comments** 🎤

### CODE REVIEW

**Review by phillsatellite — APPROVED** [review]

File: components/views/settings/pages/personalize/Personalize.html [hunk 1]
```diff
@@ -29,8 +29,8 @@
     <div class="column">
       <TypographyLabel :text="$t('pages.settings.personalize.flair')" />
       <br />
+      <!-- :disabled="theme===ThemeNames.DEFAULT" -->
```
Comment (Jekrimo) [inline components/views/settings/pages/personalize/Personalize.html:32]: Can we trash this comment?

**Review by Jekrimo — CHANGES_REQUESTED** [review]

**Review by Jekrimo — APPROVED** [review]

File: assets/styles/themes/moonless_night.less [hunk 2]
```diff
@@ -34,17 +34,6 @@
     background: @graydient !important;
   }
 
-  .flair {
```
Comment (molimauro) [inline assets/styles/themes/moonless_night.less:37]: @ThomBos all the items in this file have been moved to `base.less`?

Comment (ThomBos) [reply assets/styles/themes/moonless_night.less:37]: no, just the things in common between the default and the "moonless night" theme

Comment (molimauro) [reply assets/styles/themes/moonless_night.less:37]: good

**Review by stavares843 — CHANGES_REQUESTED** [review]
dev.satellite.one

Captura de ecrã 2022-01-19, às 19 36 08


vs this branch
Captura de ecrã 2022-01-19, às 19 35 08
Captura de ecrã 2022-01-19, às 19 35 11

it has some blue bar on the username/status sections

also the ''set photo'' copy only appears when hovering

the expected change on that screen would only be the ''set photo'' copy to be white instead of pink

you can find this screen when creating a new account

**Comment by ThomBos** [comment]
Those things are now fixed, thanks for spotting them! 🔧👀 

I've checked around the application if anything else is broken, but couldn't find any more bugs. 
Tell me whenever you find something like that! 😄

**Comment by stavares843** [comment]
those are fixed 🎉 


i noticed the blue shade on the toggles when creating a account is different compared to dev

Captura de ecrã 2022-01-20, às 18 10 45

left side is this branch, right side is dev

**Review by WanderingHogan — APPROVED** [review]

**Comment by ThomBos** [comment]
> those are fixed 🎉
> 
> i noticed the blue shade on the toggles when creating a account is different compared to dev
> 
> Captura de ecrã 2022-01-20, às 18 10 45
> 
> left side is this branch, right side is dev

The left blue should be the new primary color we are already using through the entire app 😅

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
confirmed with design as well, thanks for updating 🎉 only missing fixing the conflict

File: assets/styles/base.less [hunk 3]
```diff
@@ -187,21 +229,246 @@ button {
...
+}
 
+//Multiselect & chip items
+.multiselect {
+  .multiselect__input {
+    &:extend(.background-flair);
+    color: @text;
+  }
+  /*  &:extend(.glow-flair);
+  border: 1px solid var(--flair-color); */
```
Comment (stavares843) [inline assets/styles/base.less:285]: commented out code 🔨

File: assets/styles/base.less [hunk 4]
```diff
@@ -187,21 +229,246 @@ button {
...
+      &:extend(.background-flair);
+    }
+
+    /*  &::-moz-progress-bar {
+      &:extend(.background-flair);
+    }
+    &::-ms-fill {
+      &:extend(.background-flair);
+      border: none;
+    } */
```
Comment (stavares843) [inline assets/styles/base.less:342]: commented out code 🔨

**Comment by ThomBos** [comment]
Fixed conflict, should work now! 👍

File: assets/styles/base.less [hunk 5]
```diff
@@ -327,3 +598,67 @@ strong {
...
+      background: var(--flair-color) !important;
+      border: none;
+    }
+  }
+
+  border: 1px solid var(--flair-color);
+  background: transparent;
+  margin-bottom: @light-spacing;
+}
+ */
```
Comment (stavares843) [inline assets/styles/base.less:664]: commented out code 🔨

**Comment by stavares843** [comment]
something happened with the font color meanwhile

this branch

Captura de ecrã 2022-01-20, às 20 53 54


dev

Captura de ecrã 2022-01-20, às 20 53 59

**Comment by ThomBos** [comment]
> something happened with the font color meanwhile

if i well remember there was a ticket for this, i had to align all text colors and that white is the new one

**Comment by WanderingHogan** [comment]
@ThomBos I think you are referencing this ticket: https://satellite-im.atlassian.net/browse/AP-428 and unfortunately I think I made a mistake here.

I created that ticket to address some of the text being odd color - a bug where some of the text was pink/red had popped up.

Unfortunately, something in this branch is causing the light text color to be harder to read. Is there something you can do to address that here?
Screen Shot 2022-01-20 at 4 29 48 PM

**Comment by ThomBos** [comment]
> Unfortunately, something in this branch is causing the light text color to be harder to read. Is there something you can do to > address that here?

Sure, i can try to turn that greyer white to full white!

**Comment by ThomBos** [comment]
Should be better now, button's text are now white and more visible.
Also Liz noticed that the switches weren't aligned to the figma, so i added blur on them💥

**Comment by WanderingHogan** [comment]
Sweet, it looks really clean!

**Comment by stavares843** [comment]
neat! 🎨

**Review by stavares843 — APPROVED** [review]

**Pull request merged**